dsc/src/main.rs

```diff
@@ -187,7 +187,7 @@ Press any key to close this window
     };
 
     // MS Store runs app using `sihost.exe`
-    if parent_process.name().to_lowercase() == "sihost.exe" || parent_process.name().to_lowercase() == "explorer.exe"{
+    if parent_process.name().to_ascii_lowercase() == "sihost.exe" || parent_process.name().to_ascii_lowercase() == "explorer.exe"{
```

Comment:
What is the reason for this change?

Reply:
The crate version changed and the old method was renamed